content/code-examples/notification-templates/getOne.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Novu get single notification template
+type: snippets
+---
+
+```javascript label=Node.js
+import { Novu } from '@novu/node';
+
+export const novu = new Novu('<REPLACE_WITH_API_KEY_FROM_ADMIN_PANEL>');
+
+await novu.notificationTemplates.create("templateId");
```

Comment:
this will be  `await novu.notificationTemplates.getOne("templateId");`

Reply:
Yes, my bad.. will change